app/controllers/changeset_comments_controller.rb

```diff
@@ -1,15 +1,32 @@
 class ChangesetCommentsController < ApplicationController
+  include UserMethods
+  include PaginationMethods
+
+  layout "site", :except => :rss
+
   before_action :authorize_web
   before_action :set_locale
 
   authorize_resource
 
+  before_action :lookup_user, :only => [:index]
   before_action -> { check_database_readable(:need_api => true) }
   around_action :web_timeout
 
+  def index
+    @title = t ".title", :user => @user.display_name
+
+    comments = ChangesetComment.where(:author => @user)
+    comments = comments.visible unless current_user&.moderator?
+
+    @params = params.permit(:display_name, :before, :after)
+
+    @comments, @newer_comments_id, @older_comments_id = get_page_items(comments, [:author])
+  end
+
   ##
   # Get a feed of recent changeset comments
-  def index
```

Comment:
As per discussion at https://github.com/openstreetmap/openstreetmap-website/pull/4587 we should avoid creating new controller actions (or renaming existing ones) with non-resourceful names.

Reply:
Rebased this pull request on the first commit of [unmerged pull request](https://github.com/openstreetmap/openstreetmap-website/pull/4587) because it moves feeds out of the way.